test/e2e/managedseed/create_rotate_delete.go

```diff
@@ -89,7 +89,7 @@ var _ = Describe("ManagedSeed Tests", Label("ManagedSeed", "default"), func() {
 		}).WithPolling(5 * time.Second).Should(Succeed())
 
 		By("Wait for Seed to be ready")
-		ctx, cancel = context.WithTimeout(parentCtx, 10*time.Minute)
+		ctx, cancel = context.WithTimeout(parentCtx, 50*time.Minute)
```

Comment:
Wow, that's a big increase. What's the reason this takes so much longer? Is the testing machine running short of resources?

Reply:
I'm not sure, why this test is failing. This test case is passed in my local.

Reply:
Please revert this for now, it's probably a flake (ref https://github.com/gardener/gardener/issues/6766)